src/packagedcode/npm.py

```diff
@@ -407,15 +392,10 @@ def licenses_mapper(license, licenses, package):  # NOQA
     - (Deprecated) an array or a list of arrays of type, url.
     -  "license": "UNLICENSED" means this is proprietary
     """
-    licensing1 = get_licensing(license)
-    licensing2 = get_licensing(licenses)
-    declared_license = '\n'.join([v for v in (licensing1, licensing2) if v])
-
-    if declared_license:
-        if package.declared_license:
-            package.declared_license = '\n'.join([package.declared_license, declared_license])
-        else:
-            package.declared_license = declared_license
+    declared_license = get_declared_licenses(license)
```

Comment:
What do we do if we have both a `license` and a `licenses` attribute? This is a case that can happen. See https://github.com/nexB/scancode-toolkit/pull/1420/files#diff-44d0ea49f77a17ae876679244da9e490R37

In this case we need to return both as part of the declared license.